src/components/validation/resources/http.rs

```diff
@@ -73,36 +74,28 @@ impl HttpResourceConfig {
         }
     }
 
-    pub fn spawn_as_output(
-        self,
-        direction: ResourceDirection,
-        codec: ResourceCodec,
-        output_tx: mpsc::Sender<Vec<Event>>,
-        task_coordinator: &TaskCoordinator<Configuring>,
-        input_events: Vec<TestEvent>,
-        runner_metrics: &Arc<Mutex<RunnerMetrics>>,
-    ) -> vector_lib::Result<()> {
-        match direction {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I don't think this is needed any more with a single parameter.

Reply:
doh, forgot to remove that